core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -278,6 +279,7 @@ object KafkaConfig {
   val ReplicaFetchMaxBytesProp = "replica.fetch.max.bytes"
```

Comment:
What you say is true, but I think it's too late to add this to the KIP as we don't have a great story for deprecating properties.